data/systemd/anaconda-pre.service

```diff
@@ -9,7 +9,6 @@ Wants=rsyslog.service
 Wants=systemd-udev-settle.service
 Wants=NetworkManager.service
 Wants=plymouth-quit.service plymouth-quit-wait.service
-Wants=zram.service
```

Comment:
Should the Anaconda services depend on `zram-generator`?

Reply:
The same answer as above.